<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameSwitchButtons</class>
 <widget class="QWidget" name="FrameSwitchButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>1041</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FrameSwitchButtons</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>300</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>      风扇</string>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchBtn2" native="true">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>280</y>
     <width>350</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>0</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>57</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>节点信息</string>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchBtn1" native="true">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>140</y>
     <width>350</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchBtn3" native="true">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>420</y>
     <width>350</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchBtn4" native="true">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>560</y>
     <width>350</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchBtn5" native="true">
   <property name="geometry">
    <rect>
     <x>1510</x>
     <y>700</y>
     <width>350</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>350</y>
     <width>350</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>350</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>680</y>
     <width>231</width>
     <height>155</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>     光强</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>251</width>
     <height>155</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>     温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>211</width>
     <height>156</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>     湿度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>261</width>
     <height>155</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>二氧化碳</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>870</y>
     <width>531</width>
     <height>161</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>211</width>
     <height>123</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  氮含量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>201</width>
     <height>123</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  磷含量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>530</y>
     <width>211</width>
     <height>123</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  钾含量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>700</y>
     <width>221</width>
     <height>123</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>更新时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>150</y>
     <width>281</width>
     <height>86</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string> LED 照明   </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>440</y>
     <width>199</width>
     <height>87</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 继电器 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>580</y>
     <width>211</width>
     <height>86</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 继电器 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>710</y>
     <width>185</width>
     <height>75</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>   AI模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>602</x>
     <y>10</y>
     <width>199</width>
     <height>86</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>602</x>
     <y>563</y>
     <width>199</width>
     <height>86</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>860</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>56</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设 备 信 息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>150</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>530</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>670</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2023-7-14</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>780</y>
     <width>161</width>
     <height>107</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>160</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>720</y>
     <width>251</width>
     <height>101</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
